automation/devops_automation_infra/utils/kubectl.py

```diff
@@ -269,11 +269,12 @@ def edit_statefulset(client, name, namespace='default', new_env_var=None):
     v1_app.replace_namespaced_stateful_set(name, namespace, sts_info)
 
 
-def add_env_to_deployment(client, name, namespace='default', new_env_var=None):
+def add_env_to_deployment(client, name, namespace='default', envs=None):
```

Comment:
looks good!
you checked that the env is appended to the envs in the spec?
what will happen if i will append the same to env in that spec?

Reply:
I checked the first one, didn't try to add the same one twice, don't really make sense

Reply:
You are correct, but if the machine has not been cleaned or such a situation occurs? Just thinking about an extreme case but it really shouldn't happen